### PR TITLE
cleanup in multiphysics sources

### PR DESCRIPTION
**Pull request by trontrytel**
### Description

I unified the returned items in the named tuple. Now all the microphysics multiphysics functions return only the source terms to state variables

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: src/Atmos/Model/multiphysics_types.jl [hunk 1]
```diff
@@ -252,10 +264,10 @@ function compute_rain_snow_params(atmos, state, aux, t, ts)
         state.ρ,
     )
     S_qi -= tmp1
-    S_ρe -= state.ρ * tmp1 * (I_i + Φ)
+    S_e -= tmp1 * (I_i + Φ)
     S_qr -= tmp2
-    S_ρe += state.ρ * _L_f * tmp2
-    S_qs += (tmp1 + tmp2)
+    S_e += tmp2 * _L_f
+    S_qs += tmp1 + tmp2
```
Comment (charleskawczynski) [inline src/Atmos/Model/multiphysics_types.jl:270]: Trusting that you got this right 😄

**Review by charleskawczynski — APPROVED** [review]
On the code side, LGTM

File: src/Atmos/Model/multiphysics_types.jl (hunk 1 above)
Comment (trontrytel) [reply src/Atmos/Model/multiphysics_types.jl:270]: yep. Adding all of this to qa and actually comparing results with a paper and not my common sense is next on the list...

**Pull request merged**